lib/diagnostic.js

```diff
@@ -1,34 +1,46 @@
 'use strict'
 
 // Reuse functions wherever possible.
+const paragraph =
+  'When in the Course of human events, it becomes necessary for one ' +
+  'people to dissolve the political bands which have connected them ' +
+  'with another, and to assume among the powers of the earth, the ' +
+  'separate and equal station to which the Laws of Nature and of ' +
+  'Nature\'s God entitle them, a decent respect to the opinions of ' +
+  'mankind requires that they should declare the causes which impel ' +
+  'them to the separation.'
 
 // 1. Write a function that takes a string argument
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function () {
+  for (let i = 0; i < words.length; i++) {
+    normalizedWords[i] = words[i].toUpperCase()
```

Comment:
You're close, but you need to have your function accept a parameter `string` and then split the `string` into an array by using `string.split(' ')`